---
title: "BIO366 - Microbial Genetics"
author: "ArunK"
date: "9/2/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

[`r knitr::asis_output("\U1F332")`](https://arun-krishnaraj.github.io/evergreen)

### Notes
#### 8/26/2020
- local differences, despite global similarity; exchange between homologous chromosomes: homologous recombination
  - reciprocal exchange can be inferred using gene markers
  - gene conversion can occur when one chromosome segment is replaces
    - can occur with or without adjacent crossover; either parental configuration or novel position of flanking markers
- during mitosis, only recombination is resulting from DNA damage
- during meiosis, separation of haploid cells and creation of diploids contributes to diversity; some crossing over occurs between maternal and paternal chromosomes
  - usually reciprocal exchange between 2 chromosomes during a 4 chromosome junction
- fungi spores hold 4 distinct cells in each ascus:
  - Saccharomyces (4:4 normal segregation), and (4:4) in Ascobolus 
  
#### 8/31/2020
- Holliday Junction model for 4:4 aberrant segregation
  - identical cuts are made on paired duplexes, 'x' bond created; branch migration and extension/displacement creates 2 symmetric heteroduplexes
- Meselson-Radding model for 5:3 segregation
  - single strand cut, strand invasion and D-loop creation; D-loop chewed out, donor strand repaired using complementary strand -> single heteroduplex and resolution of Holliday junction
    - Holliday branch migration can also explain 4:4 aberrant segregation; can create symmetric heteroduplex based on Holliday junction migration direction
    
#### 9/2/2020
- Holliday junction model is usually best for explaining 4:4 aberrant selection
- Messelson is helpful for explaining 5:3, despite being less realistic; includes creation of Holliday junction and asymmetric heteroduplex to explain aberrant 4:4 selection as well
- 6:2 aberrant segregation: 
  - Initial double strand cut, followed by rescetion in the 5' to 3' and 3' to 5' directions. 5' ends are chewed faster than the 3' ends, leading to 3' overhangs; paired duplex initiates recombination using complementary sequences
  - Free 3' overhang invades paired duplex, forms D-loop; D-loop expansion and complementary synthesis; ligase reactions connect heteroduplex regions
  - Formation of a double flanking Holliday junctions; results in 6:2 aberrant selection; Holliday junction migration directions can explain both 5:3 and 4:4 aberrant selection
  - Most eukaryote germ cell recombination is initiated by double strand breaks; intentional damaging during germ cell formation contributes to genetic variation 
- Double strand break recombination verified by observation of homologous proteins that perform necessary double strand break repair functions
- Synthesis dependent strand annealing: variation of double strand gap repair; D-loop is resolved by a helicase unwinding the hybridized section, new 3' overhangs hybridize and continue complementary synthesis (6:2 in heteroduplex region) 
- Single strand DNA annealing model: double strand cut repaired in a single duplex; repeat sequences in higher eukaryotes used to hybridize after chewing and repair using complementary synthesis. Usually results in deletions due to microhomology
  - Multiple repeats can result in repeat contraction or deletion
  
#### 9/9/2020
- Holliday junction resolution: 
  - crossed phosphodiester bonds are severed, new ligases seal the nicks to produce parental configuration (same pair of strands used for exchange and resolution)
  - non-crossed strands can be severed and crossed to create 2 crossed phosphodiester pairs to produce cross over configuration (different pairs of strands used for exchange and resolution)
- double strand break repair creates 2 Holliday junctions which can resolve in the parental or cross-over configurations
  - both junctions need to resolve in the same way (inner or outer resolving, creating parental or cross-over products) in order to result in parental flanking configuration; otherwise there is cross-over of flanking markers
- X form Holliday junction can shift into isomeric H form
  - repulsion of phosphodiester backbones can force the arms away from each other, results in a square planar junction
    - we can then model resolution of parental and crossover configurations by cutting pairs of phosphodiester linkages
    - certain proteins can bind to the square planar form of junction and shift the lengths of the arms to result in heteroduplex or homoduplex creation
  
#### 9/14/2020
- site specific recombination involves specific DNA regions, usually short sequence runs 
  - minimal homology requirements
  - 4 subunits of recombinase proteins carry out the reaction
  - accessory proteins may be involved
  - no exogenous energy source required
  - DNA strands are broken and joined by transesterification chemistry
  - no degradation of DNA
- families of site-specific recombinases (mostly found in prokaryotes, except yeast: holds recombinase genes on plasmid)
  - Tyrosine family, Serine family (both side chains have hydroxyl groups), requires 4 subunits
- most site specific recombinations include 3-4 turns of DNA flanked by recombinase binding elements
  - recombinase binding elements are pallindromic, recombination target site is not; this asymmetry is used to generate a left to right notation

#### 9/16/2020
- Tyrosine recombination has antiparallel configuration: H junction formation, direct antiparallel form
  - Serine recombination is parallel: direct X crossover and resolution
- site specific recombination can trigger certain developmental pathways
- hydrolysis uses water to break phosphate in phosphodiester backbone
  - creates 3' hydroxyl and 5' phosphate ending segments
- recombinase uses hydroxyl group of serine or tyrosine as nucleophile; no loss of energy occurs 
  - creates a 3' hydroxyl and 5' phosphate esterified to serine
  - tyrosine creates a 3' phosphate esterified to tyrosine and 5' hydroxyl
- 4 serine recombinase subunits make 2 concerted breaks, rotate to bring opposing strands into alignment, then hydroxyl groups knock subunits off to complete recombination
  - 3 binding sites for serine dimers force the catalytic region into proper parallel orientation; II and III are regulatory for I unit 
- tyrosine recombinase creates a Holliday junction (antiparallel geometry), resolved by outer cuts
  
#### 9/23/2020
- recombination can occur between 2 linear or circular strands
- intramolecular reactions
  - tail head repeat segments result in excision of central region, creates a repaired sequence (excision/deletion/resolution)
- inversion occurs when a head-head sequence is resolved with cutting, flipping, and ligation
- recombinases have no way of telling head-head or head-tail configuration
  - they recognize specific sites and as a result modify global structure (depending on if deletion or inversion occur)
    - site orientation determines outcome: head-tail $\rightarrow$ deletion, head-head $\rightarrow$ inversion
      - head-tail: Tyrosine requires n (crossings) zero or even, Serine requires n odd
      - head-head: Tyrosine requires n odd, Serine requires n zero or even
- $\lambda$ phage is able to integrate into E. Coli using tyrosine recombinase
  - core site + arm sites are the target for the phage
  - lysis and excision is a direct reversal of the initial integration (controlled by Xis, determines which direction is undertaken)
  
#### 9/28/2020
- intramolecular recombination: HIN, results in inversion by site-specific recombination
  - H segment inverted in Salmonella (adjusts flagellum expression, avoids host immune response), consists of a promoter region that is used to swap between flagellum types
- Gin recombinase: modifies tail fiber to adjust infection capabilities; inversion results in slight changes to the variable invertible region (codes for 2 types of receptor recognition)  
- transposon resolution: free to jump in and out of genomes, can cointegrate and add an extra copy of the coding region
  - XerC: ensures that chromosomal copies are properly segregated and resolved to form two circular genomes
- yeast carries a specific plasmid; resides in nucleus, neutral to host, carries 40-60 copies without any detrimental effects
  - optimized to use specific host replication machinery to generate progeny 
  - FLP recombination site (tyrosine recombinase), results in crossover following bidirectional replication initiation 

#### 9/30/2020
- site specific combination usually uses Flp and Cre; versatile, can result in excision or insertion depending on site orientation
  - GFP is used to generate recombinant chromosomes with promoter, target site, and GFP gene
  - recombination sites can be used to excise genes of interest and observe traits; can also be used to turn on promoter regions to test genes
    - transcription terminator can be removed to result in complete promoter gene region
- recombination sites require complementary base pair recombinant sites (native-native or altered-altered recombination)
  - we can modify Flp and Cre to recognize desired recombination sites by generating mutants; can then select for desired Flp mutants with galactosidase coloring of E Coli in X gal medium (looking for red Flp variant colonies)
  
#### Section 1 Reading
Homologous Genetic Recombination  

- Recombination: process (almost uniform) of reshuffling
- diploid organisms contain two copies of each chromosome (autosome)    
  - sperm and egg pick up meiotic recombination and form new combinations  
- Homologous recombination: exchange between partners with high sequence homology (contrasted to site-specific recombination)
  - initiation, strand exchange, branch migration, heteroduplex creation and resolution
- How does recombination occur in haploid bacteria?
  - sister chromosomes can be created , but aren't useful for creating variation unless mutations are introduced
    - useful for repair (double strand, or replication errors)
  - use of intact chromosome during transformation; DNA from dead bacterium creates a diploid (merodiploid); DNA transfer by conjugation, transduction, and transformation can can create E. Coli Merodiploids
- Linear duplex exchange results in linear molecules
  - circular chromosomes exchange into a single large circle (or remain individual if partnered circular chromosomes)
  - viable exchange between circular and linear molecules (bacterial chromosome and DNA segment) requires a minimum of 2 exchanges to result in desired homologous recombination
- Reciprocal exchange involves local nonhomologous segments swapping between DNA partners during meiosis
- Gene conversion occurs when an exchange event replaces a nonhomologous segment (unidirectional information transfer); markers flanking the transferred segment can remain or swap to generate parental or cross-over marker configurations
- Saccharomyces & Ascobolus are useful for studying meiosis, since they pack products of meiosis into an ascus (sampling meitotis events vs tetrad analysis)
  - following two homologous chromosomes replicating, and resolving exchanges, in Saccharyomyces the four duplexes are packaged into spores, then into a spore sack; each spore generates a colony
  - following meitotic recombination in Ascobolus, the four duplexes replicate again to result in 8 total spores in the ascus (4 sister spore pairs)
- Saccharoymyces segregation patterns:
  - No exchange, 2 duplexes carry M, 2 carry m -> 2:2 normal segregation
    - could result if additional events reverse exchanges 
  - 2:2 aberrant, 1 M, 1 m, 2 half-sectored colonies -> 2:2 (aberrant)
  - $2\frac{1}{2}$:$1\frac{1}{2}$, 2 M (or m), 1 m (or M), 1 half-sectored colony
  - 3:1, 3 M (or m) colonies, 1 m (or M) colony
- Ascobolus segregation patterns:
  - 4:4, 4 M colonies, 4 m colonies
  - 4:4 aberrant, 1 M, m sister pair, 2 non-identical sister pairs
  - 5:3, single non-identical sister pair (m+M)
  - 6:2, all identical sister spores, but 3 pairs of 1 phenotype with 1 pair of the other
Models of Recombination
- Holliday Junction model
  -I Holliday Junction formation
  -II Branch migration (right movement of 'x'), creation of symmetric heteroduplex
  -III Same junction as II, shift from 'x' form to 'H' form, antiparallel geometry
- Manipulation of the Holliday Junction
  - junction 'x' is resolved by cutting crossed thin strands; creation of symmetric heteroduplexes with parental configurations (Ab, aB)
  - junction type 'Y' is resolved by cutting crossed thick strands; creation of symmetric heteroduplex, crossed-over marker configurations
- Meselson-Radding Model
  - strand nick made in one duplex, which then invages the recipient 
    - -> D-loop creation; heteroduplex extension & D-loop expansion; chewing away of D-loop and close of nicks, Holliday Junction at end of heteroduplex (heteroduplex is present on one duplex only)
- Double Strand Gap Repair Model:
  - recipient strands generate 3' extensions via unequal degradation; 3' extensions invade heteroduplex to create a D-loop
    - creation of two Holliday Junctions; results in all-donor region, and two asymmetric heteroduplex on either side