---
title: "BIO366 - Microbial Genetics"
author: "ArunK"
date: "9/2/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

[`r knitr::asis_output("\U1F332")`](https://arun-krishnaraj.github.io/evergreen)

### Notes
#### 8/26/2020
  - reciprocal exchange can be inferred using gene markers
  - gene conversion can occur when one chromosome segment is replaces
    - can occur with or without adjacent crossover; either parental configuration or novel position of flanking markers
- during mitosis, only recombination is resulting from DNA damage
- during meiosis, separation of haploid cells and creation of diploids contributes to diversity; some crossing over occurs between maternal and paternal chromosomes
  - usually reciprocal exchange between 2 chromosomes during a 4 chromosome junction
- fungi spores hold 4 distinct cells in each ascus:
  - Saccharomyces (4:4 normal segregation), and (4:4) in Ascobolus 
  
#### 8/31/2020
- Holliday Junction model for 4:4 aberrant segregation
  - identical cuts are made on paired duplexes, 'x' bond created; branch migration and extension/displacement creates 2 symmetric heteroduplexes
- Meselson-Radding model for 5:3 segregation
  - single strand cut, strand invasion and D-loop creation; D-loop chewed out, donor strand repaired using complementary strand -> single heteroduplex and resolution of Holliday junction
    - Holliday branch migration can also explain 4:4 aberrant segregation; can create symmetric heteroduplex based on Holliday junction migration direction
    
#### 9/2/2020
- Holliday junction model is usually best for explaining 4:4 aberrant selection
- Messelson is helpful for explaining 5:3, despite being less realistic; includes creation of Holliday junction and asymmetric heteroduplex to explain aberrant 4:4 selection as well
- 6:2 aberrant segregation: 
  - Initial double strand cut, followed by rescetion in the 5' to 3' and 3' to 5' directions. 5' ends are chewed faster than the 3' ends, leading to 3' overhangs; paired duplex initiates recombination using complementary sequences
  - Free 3' overhang invades paired duplex, forms D-loop; D-loop expansion and complementary synthesis; ligase reactions connect heteroduplex regions
  - Formation of a double flanking Holliday junctions; results in 6:2 aberrant selection; Holliday junction migration directions can explain both 5:3 and 4:4 aberrant selection
  - Most eukaryote germ cell recombination is initiated by double strand breaks; intentional damaging during germ cell formation contributes to genetic variation 
- Double strand break recombination verified by observation of homologous proteins that perform necessary double strand break repair functions
- Synthesis dependent strand annealing: variation of double strand gap repair; D-loop is resolved by a helicase unwinding the hybridized section, new 3' overhangs hybridize and continue complementary synthesis (6:2 in heteroduplex region) 
- Single strand DNA annealing model: double strand cut repaired in a single duplex; repeat sequences in higher eukaryotes used to hybridize after chewing and repair using complementary synthesis. Usually results in deletions due to microhomology
  - Multiple repeats can result in repeat contraction or deletion
  
#### 9/9/2020
- Holliday junction resolution: 
  - crossed phosphodiester bonds are severed, new ligases seal the nicks to produce parental configuration (same pair of strands used for exchange and resolution)
  - non-crossed strands can be severed and crossed to create 2 crossed phosphodiester pairs to produce cross over configuration (different pairs of strands used for exchange and resolution)
- double strand break repair creates 2 Holliday junctions which can resolve in the parental or cross-over configurations
  - both junctions need to resolve in the same way (inner or outer resolving, creating parental or cross-over products) in order to result in parental flanking configuration; otherwise there is cross-over of flanking markers
- X form Holliday junction can shift into isomeric H form
  - repulsion of phosphodiester backbones can force the arms away from each other, results in a square planar junction
    - we can then model resolution of parental and crossover configurations by cutting pairs of phosphodiester linkages
    - certain proteins can bind to the square planar form of junction and shift the lengths of the arms to result in heteroduplex or homoduplex creation

#### Section 1 Reading
Homologous Genetic Recombination  

- Recombination: process (almost uniform) of reshuffling
- diploid organisms contain two copies of each chromosome (autosome)    
  - sperm and egg pick up meiotic recombination and form new combinations  
- Homologous recombination: exchange between partners with high sequence homology (contrasted to site-specific recombination)
  - initiation, strand exchange, branch migration, heteroduplex creation and resolution
- How does recombination occur in haploid bacteria?
  - sister chromosomes can be created , but aren't useful for creating variation unless mutations are introduced
    - useful for repair (double strand, or replication errors)
  - use of intact chromosome during transformation; DNA from dead bacterium creates a diploid (merodiploid); DNA transfer by conjugation, transduction, and transformation can can create E. Coli Merodiploids
- Linear duplex exchange results in linear molecules
  - circular chromosomes exchange into a single large circle (or remain individual if partnered circular chromosomes)
  - viable exchange between circular and linear molecules (bacterial chromosome and DNA segment) requires a minimum of 2 exchanges to result in desired homologous recombination
- Reciprocal exchange involves local nonhomologous segments swapping between DNA partners during meiosis
- Gene conversion occurs when an exchange event replaces a nonhomologous segment (unidirectional information transfer); markers flanking the transferred segment can remain or swap to generate parental or cross-over marker configurations
- Saccharomyces & Ascobolus are useful for studying meiosis, since they pack products of meiosis into an ascus (sampling meitotis events vs tetrad analysis)
  - following two homologous chromosomes replicating, and resolving exchanges, in Saccharyomyces the four duplexes are packaged into spores, then into a spore sack; each spore generates a colony
  - following meitotic recombination in Ascobolus, the four duplexes replicate again to result in 8 total spores in the ascus (4 sister spore pairs)
- Saccharoymyces segregation patterns:
  - No exchange, 2 duplexes carry M, 2 carry m -> 2:2 normal segregation
    - could result if additional events reverse exchanges 
  - 2:2 aberrant, 1 M, 1 m, 2 half-sectored colonies -> 2:2 (aberrant)
  - $2\frac{1}{2}$:$1\frac{1}{2}$, 2 M (or m), 1 m (or M), 1 half-sectored colony
  - 3:1, 3 M (or m) colonies, 1 m (or M) colony
- Ascobolus segregation patterns:
  - 4:4, 4 M colonies, 4 m colonies
  - 4:4 aberrant, 1 M, m sister pair, 2 non-identical sister pairs
  - 5:3, single non-identical sister pair (m+M)
  - 6:2, all identical sister spores, but 3 pairs of 1 phenotype with 1 pair of the other
Models of Recombination
- Holliday Junction model
  -I Holliday Junction formation
  -II Branch migration (right movement of 'x'), creation of symmetric heteroduplex
  -III Same junction as II, shift from 'x' form to 'H' form, antiparallel geometry
- Manipulation of the Holliday Junction
  - junction 'x' is resolved by cutting crossed thin strands; creation of symmetric heteroduplexes with parental configurations (Ab, aB)
  - junction type 'Y' is resolved by cutting crossed thick strands; creation of symmetric heteroduplex, crossed-over marker configurations
- Meselson-Radding Model
  - strand nick made in one duplex, which then invages the recipient 
    - -> D-loop creation; heteroduplex extension & D-loop expansion; chewing away of D-loop and close of nicks, Holliday Junction at end of heteroduplex (heteroduplex is present on one duplex only)
- Double Strand Gap Repair Model:
  - recipient strands generate 3' extensions via unequal degradation; 3' extensions invade heteroduplex to create a D-loop
    - creation of two Holliday Junctions; results in all-donor region, and two asymmetric heteroduplex on either side

  
#### 9/14/2020
- site specific recombination involves specific DNA regions, usually short sequence runs 
  - minimal homology requirements
  - 4 subunits of recombinase proteins carry out the reaction
  - accessory proteins may be involved
  - no exogenous energy source required
  - DNA strands are broken and joined by transesterification chemistry
  - no degradation of DNA
- families of site-specific recombinases (mostly found in prokaryotes, except yeast: holds recombinase genes on plasmid)
  - Tyrosine family, Serine family (both side chains have hydroxyl groups), requires 4 subunits
- most site specific recombinations include 3-4 turns of DNA flanked by recombinase binding elements
  - recombinase binding elements are pallindromic, recombination target site is not; this asymmetry is used to generate a left to right notation

#### 9/16/2020
- Tyrosine recombination has antiparallel configuration: H junction formation, direct antiparallel form
  - Serine recombination is parallel: direct X crossover and resolution
- site specific recombination can trigger certain developmental pathways
- hydrolysis uses water to break phosphate in phosphodiester backbone
  - creates 3' hydroxyl and 5' phosphate ending segments
- recombinase uses hydroxyl group of serine or tyrosine as nucleophile; no loss of energy occurs 
  - creates a 3' hydroxyl and 5' phosphate esterified to serine
  - tyrosine creates a 3' phosphate esterified to tyrosine and 5' hydroxyl
- 4 serine recombinase subunits make 2 concerted breaks, rotate to bring opposing strands into alignment, then hydroxyl groups knock subunits off to complete recombination
  - 3 binding sites for serine dimers force the catalytic region into proper parallel orientation; II and III are regulatory for I unit 
- tyrosine recombinase creates a Holliday junction (antiparallel geometry), resolved by outer cuts
  
#### 9/23/2020
- recombination can occur between 2 linear or circular strands
- intramolecular reactions
  - tail head repeat segments result in excision of central region, creates a repaired sequence (excision/deletion/resolution)
- inversion occurs when a head-head sequence is resolved with cutting, flipping, and ligation
- recombinases have no way of telling head-head or head-tail configuration
  - they recognize specific sites and as a result modify global structure (depending on if deletion or inversion occur)
    - site orientation determines outcome: head-tail $\rightarrow$ deletion, head-head $\rightarrow$ inversion
      - head-tail: Tyrosine requires n (crossings) zero or even, Serine requires n odd
      - head-head: Tyrosine requires n odd, Serine requires n zero or even
- $\lambda$ phage is able to integrate into E. Coli using tyrosine recombinase
  - core site + arm sites are the target for the phage
  - lysis and excision is a direct reversal of the initial integration (controlled by Xis, determines which direction is undertaken)
  
#### 9/28/2020
- intramolecular recombination: HIN, results in inversion by site-specific recombination
  - H segment inverted in Salmonella (adjusts flagellum expression, avoids host immune response), consists of a promoter region that is used to swap between flagellum types
- Gin recombinase: modifies tail fiber to adjust infection capabilities; inversion results in slight changes to the variable invertible region (codes for 2 types of receptor recognition)  
- transposon resolution: free to jump in and out of genomes, can cointegrate and add an extra copy of the coding region
  - XerC: ensures that chromosomal copies are properly segregated and resolved to form two circular genomes
- yeast carries a specific plasmid; resides in nucleus, neutral to host, carries 40-60 copies without any detrimental effects
  - optimized to use specific host replication machinery to generate progeny 
  - FLP recombination site (tyrosine recombinase), results in crossover following bidirectional replication initiation 

#### 9/30/2020
- site specific combination usually uses Flp and Cre; versatile, can result in excision or insertion depending on site orientation
  - GFP is used to generate recombinant chromosomes with promoter, target site, and GFP gene
  - recombination sites can be used to excise genes of interest and observe traits; can also be used to turn on promoter regions to test genes
    - transcription terminator can be removed to result in complete promoter gene region
- recombination sites require complementary base pair recombinant sites (native-native or altered-altered recombination)
  - we can modify Flp and Cre to recognize desired recombination sites by generating mutants; can then select for desired Flp mutants with galactosidase coloring of E Coli in X gal medium (looking for red Flp variant colonies)
  
#### MT2 Review
- homologous recombination requires stretches of homology, while site-specific recombination requires minimal
- site specific recombination results in no change to target sites, but rearrangements to flanking DNA
  - conservative site specific recombination works through transesterification to active subunits, and as a result requires no input of energy
- phosphodiester bonds in DNA can undergo nucelophilic attack by serine, tyrosine or water
- phosphodiester hydrolysis can result in two products depending on the location of the break (3' OH or 5' OH)
- serine recombination is parallel strand orientation, while tyrosine recombination is antiparallel orientation
- 4 subunits of recombinase carry out reaction; potentially includes accessory proteins
  - no degradation or synthesis of new DNA
- tyrosine recombinase family   
  - $\lambda$ integrase, Flp, Cre, XerC/XerD
  - uses 4 monomers, two per site; (two-step) single strand cleavage, transesterification and Holliday junction formation, resolution and exchange
  - alignment state crossings 0 or even: deletion; odd: inversion
- serine recombinase family
  - Hin invertase, Tn3 resolvase, Gin invertase
  - uses 2 dimers, creates double strand breaks and covalent linkage to subunits; complex rotates and resolves with hydroxyl acting as a nucleophile
  - alignment state crossings odd: deletion; 0 or even: inversion
- head-tail sites generate excision, head-head sites generate inversion
- integration and excision of strands can occur reversibly with the same recombinase
- phase variation utilizes inversion recombination to avoid host immune detection
  - variable expression of 2 flagellum tail proteins 
- selfish yeast plasmid: maintains number of copies per daughter cell via amplification
  - bifurcated rolling circle spin-off mechanism to reach 60 copies per cell
  ![](/Users/arunk/Documents/GitHub/arun-krishnaraj.github.io/images/plasmid_amp.JPG)
    
#### Section 2 Reading
Site-specific recombination

- targets shorter DNA sites than homologous recombination
  - also usually carried out by a small number of proteins 
- conservative site-specific recombination retains the high energy held in the bonds, by using a transesterification mechanism; present almost exclusively in prokaryotes
  - invertase family (serine family): name based on mediation DNA inversion or resolution
    - aligns target sites in parallel geometry, double strand cuts sites and become covalently linked; subunit rotation and resolution
    ![](/Users/arunk/Documents/GitHub/arun-krishnaraj.github.io/images/invertase.JPG)
    - resolvase target site has 3 sub-sites, 2 of which are used to pull one site into proper conformation
  - lambda integrase family (tyrosine family): name based on mediation genome integration
    - creates single strand breaks, forms intermediate Holliday junction; antiparallel arrangement of segments
      ![](/Users/arunk/Documents/GitHub/arun-krishnaraj.github.io/images/tyrosine.JPG)
  - both use two subunits flanking each site (head-head)

Structural support of Recombination Mechanism

  - intermediate of resolvase recombination suggests resolvase must undergo substantial conformational change to recombine strands
  - tyrosine recombination involves only 2 of 4 active sites at once (causes exchange to occur in two separate steps; Holliday formation, junction isomerization and resolution)

Biological consequences of site-specific recombination

- site-specific recombination can occur between sites on two separate DNA molecules, or sites on a single DNA molecule
- recombination between direct sites results in excision; recombination between inverted sites will result in DNA inversion (both only act on sites on a single DNA molecule)
- lambda can integrate and excise using the same tyrosine recombinase in conjuction with host IHF and Fis; determines whether phage remains lysogenic or enters lytic phase
      ![](/Users/arunk/Documents/GitHub/arun-krishnaraj.github.io/images/lambda.JPG)
- Salmonella uses site-specific recombination to avoid host antibodies attacking flagellum
  - phase variation: switching between flagellin subunit displayed
  - mediated by serine recombinase switching on and off repressor segments
- transposons can move within locations in a genome or between genomes
  - Tn3 is a replicative transposon: duplicates itself when excised
- Flp recombinase is used to amplify yeast plasmids; restores stable number of plasmid in daughter cells via amplication

Applications of site-specific recombination

- DNA recombination can be utilized to generate integration, excision or inversion of DNA - Cre and Flp are commonly used for gene editing
- cell lineage can be monitored using Flp and fluorescent protein-coding regions 
- Flp can be used to remove a gene by flanking it with binding sites
- genes can be turned on using non-functional orientation of promoter regions or by excising a terminator site between promoter and gene regions
- we can generate altered binding site recombinases that are not cross-reactive

#### 10/05/2020
- mutant recombinases can be used to generate multiple recombinant genomes; difficult to screen enough colonies + time consuming to identify mutants
  - use of fluorescent proteins instead to quickly sort recombinants; tyrosine recombination results in inversion and shift from yellow to green fluorescent proteins 
- directed genome editing can be used to introduce native genomes to repair targeted mutations
- zinc fingers: first discovered as a transcription factor, recognizes 3 base pairs (5'-3') at residues 2, 3, 6
  - consists of two $\beta$ sheets, 2 loops, and $\alpha$ helix
  - can use multiple ZnFs in a single module to increase specificity, reduced odds of the same sequence occurring by chance 
  - addition of nuclease (restriction enzyme based) to zinc finger to cause double strand break; split recognition specificity into two zinc + nuclease modules
  
#### 10/07/2020
- TALE: transcription activator like effector, bind to promoters and activate/repress gene products to establish infection
  - recognizes a single base pair, unlike zinc finger
  - consists of dual $\alpha$ helix repeats, specific region repeat codes for nucleotide binding motifs
  - recognizes N to C 5'-3', opposite of Zinc finger recognition arrangement
  
#### 10/14/2020
- generate a series of 3 unit zinc finger modules with altered middle finger specificity; enables sufficient binding to target site
 - recognition of sites is determined by individual fingers, overall strength is determined by number of fingers used
- phage display: engineered proteins can be displayed on phage coat, then selected and enriched in subsequent infections 
- use of biotin affinity tag to pull oligonucleotide to streptavidin 
- use wild type oligio as competitor, originally unlinked without biotin; wild oligo competes for phage binding, and oligo of interest are pulled out of solution by avidin
  - repeated enrichment furthers specificity and affinity; first removes wild, next removes unwanted trinucleotide binding phages (15), then once again with 64 random trinucleotide primers (64)
- CRISPR: form of prokaryotic adaptive immune memory, defense against foreign nucleic acids (invading DNAs)
  - target DNA is cut to trigger degradation

#### 10/19/2020
- almost all prokaryotes contain a CRISPR system for adaptive immune memory; protects against foreign nucleic acids, using an RNA based interference complex
- there are multiple types of CRISPR systems, depending on the target for degradation;  general locus is leader-repeat-spacer-repeat-spacer-repeat-spacer-repeat
  - class I uses multiple Cas proteins, while class II uses a single Cas protein; we focus on class II
  - spacer sequence cut from invading nucleic acid and contributes to immune memory; acquisition -> CRIPSR RNA biogenesis -> interference 
    - CRISPR RNA associates with future infecting nucleic acids, and Cas proteins destroy the complex 
    - repeat segments are used to excise spacers by site-specific recombination
  - Cas3 has helicase-DNase activity, moving along single stranded DNA paired to CRISPR RNA
  - Type 1 CRISPR systems utilize Cas 1, 2 to acquire spacer segments (canonical PAM AAG-TTC); cut protospacer sequence following the PAM, inserted into the CRISPR locus near the leader sequence 
  - repeat segments are symmetric, RNA forms stem-loops; each crRNA differs only at the spacer sequence, then form cascade complex to recognize target RNA
    - spacers bind to complementary invading DNA, forms D-loop which is then excised 
- Type II CRISPR system: use a single Cas9 protein, with multiple RNAs (crRNA and tracrRNA)
  - crRNA and tracrRNA can be fused to form a single guide RNA (sgRNA)
  - PAM sequence can vary based on sequence and Type of system (3' side of protospacer, cut 2 from the PAM instead of 5' and immediately after PAM in Type I)
  - tracrRNA hybridizes to repeat sequences in crRNA, RNase III splits individual crRNAs
    - tracrRNA remains hybridized to repeat sequence to form final interference RNA in Cas9 system
    - spacer binds to complementary PAM strand, Cas9 double-strand cuts 2 bases from the PAM site; can then be used for recombination and repair 

#### 10/21/2020
- Type III CRISPR systems can target DNA and RNAs, without targeting of PAM sequences during acquisition or deletion targeting; Csm complex targets DNA, while Cmr complex targets RNA
  - coupled with transcription of invading nucleic acid
  - CRISPR array is transcribed then processed by Cas9 to form crRNAs; then assembled into Csm/Cmr and matured at the 3' repeat end
    - repeat hybridizes to RNA transcribed from invading sequence; once hybridized, complex can degrade RNA being produced or looped out DNA
  - flanking sequence is used to differentiate between self and non-self sequences (protospacer adjacent spacer sequence, does not bind to target strands)
- acquisition phase for CRISPR is well conserved
  - Cas1 and Cas2: catalytic activity, DNA transposase and accessory protein respectively (4:2)
    - other proteins like Csn2 and Cas9 may also play a role in targeting 
  - protospacer is cut without PAM, then joined to leader-repeat junction and repeat-spacer junction (result in leader-repeat-spacer-repeat-spacer)
  - protospacer uses two free 3' hydroxyls in two nucleophilic attacks to fully integrate; gaps are then filled in, duplication of disrupted spacer sequence 
- sgRNA forms tetraloop to generate immunity complex for disruption step 
  
#### 10/26/2020
- CRISPR Type II structure
  - target DNA hybridizes to guide sequence in sgRNA (tetraloop consisting of loops and antirepeats)
  - we are interested in supressing NHEJ in repair of CRISPR-induced double strand breaks
- traffic light reporter system: shows how much of repair is HDR vs NHEJ
  - uses red and green reporters to detect NHEJ or HDR and cell sorter
  - alternative reading frames resulting from insertions or deletions remove GFP expression; frameshift removes stop codon, allowing transcription of RFP
  - default reporter created by inserting target sequence to shift reading frame, no GFP or RFP

#### Section 3 Readings

Zinc Finger Nucleases

- directed genome editing takes advantage of existing repair mechanisms to solve induced damage
- Zinc fingers are restriction enzymes used to restrict invasion of nucleic acids; specific 4-6 bp sequences are recognized, and double strand breaks are made
  - large number of susceptible sites in a genome 
  - Fokl: restriction enzyme which cuts at a distance from the binding site, with separate DNA binding and cleaving domains
  - Zinc finger domains: individual zinc fingers recognize 3 nucleotides, sequence range is made of 3 finger subunits; two sets of zinc fingers must be used, making the binding specific to an 18-mer
    - ZFNs have high specificity, low chance of off-site recognition
  - selection schemes:
    - phage display: Zinc finger peptides are cloned into bacteriophage, which is then amplified in a bacterial host; use oligionucleotides of the target motif to isolate phages (biotin streptavidin interaction); control oligionucleotides are included to reduce random binding 
  - Fokl monomers require ZFNs with binding sites spaced 5 to 6 bp apart

TALENs (Transcription Activator-Like Effector Nucleases)

- TALEs are derived from plant pathogen Xanthomonas genera; proteins bind to promoters in host genome to regulare transcription, useful for establishing infection
  - host can also utilize effector proteins to initiate immune response
  - nucleotides 12 and 13 vary in effector protein modules, represent specific recognition of individual base pairs
  - TALE repeat modules have right-handed chirality, and can make contact in the major DNA groove; alignment N-C with 5'-3' of contact DNA
  - two TALEN subunits are engineered to have recognition sites separated enough for Fokl nuclease to dimerize 

Specificity Selection 

- use of 3 zinc fingers per module, two modules which recognize opposing strands of DNA (6-10 bp separation enables Fokl dimerization)
  - specificity can be generated from base zinc fingers `5'GNN3'`,`5'ANN3'`,`5'CNN3'` and `5'TNN3'`
  - after mutagenesis, we perform competitive selection with parental motif, alterative 15 trinucleotides, and random 64 trinucleotides 
  
Traffic Light Reporter

- CHYSEL: cis-acting hydrolase element, transcribed by ribosome but does not form a peptide bond (ribosome skipping mechanism produces two separate proteins)
  - inserted between GFP and RFP, if unread produces GFP with stop codon
  - if in correct reading frame, GFP is gibberish, and RFP is transcribed as a separate protein
  - GFP(+1) + T2A(+1) + RFP(+1): +1 ORF produces functional GFP and RFP, and CHYSEL splits them to produce cells expressing both
  - GFP(+1) + TG + T2A(+3) + RFP(+3): +1 ORF produces GFP, followed by a stop codon; T2A and RFP are not transcribed and green fluorescence is observed (HDR)
    - +3 ORF produces no GFP, T2A is in frame and splits to produce active RFP, which is also in frame
  - GFP(+1) - NN + T2A(+3) + RFP(+1): reading frame for GFP moves from +1 to +3, producing inactive GFP, active T2A and RFP
  - NHEJ (either deletion or insertion) results in RFP, while HDR generates GFP only
  - experimental reporter:
    - GFP + Target Site + GFP + TG + T2A + RFP
      - produces no protein without strand break due to stop codon 
      - HDR: using GFP repair template, target site is removed by gapping; repaired reporter is GFP(+1) + CHYSEL(+3) + RFP(+3) in ORF +1
      - NHEJ: partial deletion, produces GFP(+1 to +3) + CHYSEL(+3) + RFP(+3) in ORF +3
- general ORF shifting rules
  - insertion of n: n steps down (ins1 = +1 -> +3)
  - deletion of n: n steps up (del2 = +1 -> +3), middle deletions result in partial ORFs (mdel2 = +1 -> +1:+3)
  
CRISPR

- originally bacterial adaptive immune systems, defense against invading nucleic acids
  - detected by base-pairing with crRNA and bound proteins 
  - thought to control expansion of genome by foreign DNA, could explain difference in presence of nonfunctional DNA in eukaryotic vs prokaryotic genomes
- protospacer sequence is excised from invading nucleic acid, and inserted into the CRISPR locus
  - recognition by PAM motifs, which are removed prior to integration
  - integration duplicates flanking repeat segments, mediates the total number of spacers able to be held
- CRISPR locus is transcribed then processed into smaller crRNAs (complementary to spacers), with partially modified repeat segments
  - Type I has PAM proximal to 5' end of looped-out DNA
    - crRNAs generated by Cas6 endonuclease activity
    - crRNA forms a complex, then recognizes foreign DNA; Cas3 (helicase-nuclease) is recruited, cleaves the sDNA; DNA-RNA duplex is unwound
  - Type II has PAM proximal to 3' end of looped-out DNA
    - crRNAs generated by trans-acting RNA tracrRNA
    - single protein assocates with crRNA, Cas9 associates with dual heteroduplex RNA (tracrRNA + crRNA)
  - Type III
    - crRNA forms either Csm or Cmr complexes to target invading DNA or viral RNA respectively
- CRISPR-Cas9 for genome editing
  - crRNA can be selected for any DNA sequence of interest, tracrRNA can be produced along with Cas9
    - double strand break is caused at the target locus, repair can be NHEJ or HDR
- Spacer acquisition 
  - Cas1 and Cas2 form a complex (4:2), invading protospacer is excised, 3' hydroxyl serves a nucleophile at the spacer nearest to the leader sequence; resolution occurs as 3' hydroxyl acts a nucleophile
    - insertion of spacer and ligation creates duplicate flanking repeats
  
#### 10/28/2020
- Jiang et al. paper
  - uses Pneumococcus as experimental organism
    - high frequency of homologous recombination and transformation (also used in Avery-Griffith experiment)
  - experimental design
    - insert CRISPR component gene, and the target DNA into another chromosomal locus; thus investigating the mechanistic basis for CRISPR action in bacteria 
    - target site may mutate to become a poor target for CRISPR binding, use of KanR marker to ensure genetic transfer 
  - we expect to only see colonies if the target is not susceptible (CRISPR cannot kill the strain without a proper binding site)
    - we get 10x more KanR colonies in the CRISPR resistant strain than the susceptible strain  
    - due to recombination effects, able to repair CRISPR susceptible strain about 1 in 100 times; we can amplify resistant locus to replace susceptible locus, more KanR colonies
  - use of mutant PAM sequences ($4^5$ combinations), only non-targeted PAM sequences survive and gain chloramphenicol resistance; resulting lines are sequences  
  - two distinct PAM bindings were possible NGG or NNG

#### 11/04/2020
- KanR occurs if CRISPR integrated into target strain, but tranformant SortA is not excised 
 - certain protospacer sites are more important for CRISPR binding than others (certain bases exhibit depleted sequence survival); bases 1-6 prior to PAM cannot have any mutations, otherwise no binding occurs 
- Gibson assembly: segment contains two bidirectional primer sequences, producing tracr-cas9 and aphA segments by PCR; resection to single-stranded overhangs, ligation to create complete insertion sequence
  - insertion results in recombination to introduce KanR and PAM binding mutation (CRISPR integration signaled by Kanamycin resistance, but only mutant PAM binding sequences survive)
    - edited genome has form tracr-Cas9-Spacer-KanR-SortA, only mutant SortA survives 
  - resistance loci can be swapped back and forth to sequentially edit multiple genes
- gene editing proof: drug resistance to Kan or Clor (transformed sequence required or CRISPR kills cell), gel verification of fragment size, enzymatic activity (2-3 tests to validate editing)
- Beta-galactosidase is a useful for assaying, due to clear color change
  - mutate null to no function enyzme variant, also modifies the PAM site (first modify SrtA, then bGal)
  - editing template in all treatments produces significantly more transformants than the control (point mutation, insertion frameshift, and large deletion)
    - verification by gel electrophoresis shows characteristic sized fragments
  - removing srtA shows a relative increase in supernatant enzymatic activity
  - second edit for bGal reduces total level of enzymatic activity
- an additional experiment was performed: attempt to integrate KanR, srtA and bgaA repeats, mutant srtA and bgaA in a single targeting step
  - found to be possible, but gel verification shows relatively more escapees 
- following gene editing, CRISPR locus must be excised

#### 11/09/2020
- we can use a plasmid with the CRISPR target sequence, used to verify whether the original sequence was properly used in homologous recombination 
  - transforming plasmid contains bgaA protospacer and spectinomycin resistance, select for plasmid; can then use a temperature sensitive plasmid to remove it 
- CRISPR can aid the editing process in 2 synergistic ways:
  - cause double-strand break to induce recombination with introduced template
  - CRISPR can also kill unedited cells, selection for editing cells 
- use CRISPR with KanR segment to restore normal erythromycin function, isolate in Kan Erm plates
  - transformation removes PAM sequence to stop CRISPR activity
  - Kan assays for CRISPR introduction, Erm assays for editing 
- by comparing normal and random CRISPRs, we can determine the mechanism of editing
  - both produce similar fractions of ermR colonies of total colonies 1 in 100 (frequency of homologous repair with erm locus)
    - higher fraction of dummy CRISPR kanR colonies are ermR (1 in 10); actual CRISPR has nearly all kanR colonies also ermR (unedited ermR cells were likely killed by CRISPR)
  - overall suggests that CRISPR aids editing by killing unediting cells, could also be small effect of double-strand break introduction 
  - higher number of kanR colonies for dummy CRISPR; functional CRISPR kills non-erm-edited colonies, while non-functional leaves all colonies alive after conferring KanR
    - when using no editing template, dummy CRISPR produces much less; difference is the ability of CRISPR to kill, remaining colonies are escapers (1 in 1000)
  - while frequency of editing is only 1 in 100, CRISPR is able to kill enough unedited cells to raise the proportion of edited cells to 99%; hugely increases efficiency of transformation, reduces chances of observing escapers
  - since homologous recombination can occur at the CRISPR Kan locus and the Erm Locus, we can get the total recombination frequency for CRISPR-containing cells: ErmR and KanR 
    - higher for CRISPR than dummy ($2\times10^{-4}$ vs $9\times10^ {- 5}$); recombination frequency is raised by $\sim3$, smaller effect attributed to double-strand break causation in Pneumococcus 
  
#### 11/11/2020
- difference between Pneumococucus and E Coli on efficiency of recombination (E. Coli recombination is usually accomplished by mating) 
  - usually used with electroporation, increases recombination frequency (weaker than Pneumococcus in DNA uptake and recombination)
  - recombination will produce KanR in 1 of 200 Pneumococcus; CRISPR is able to kill unedited cells, reducing total number of colonies required for screening (useful in non-rate limiting organisms)
  - CRISPR must be brought into E. Coli as a plasmid (linear DNA can only be amplified/stable if homologously incorporated into DNA), which can be stable outside chromosomes
- CRISPR introduced in the plasmid, editing of ribosomal protein
  - ribosome will become resistant to Streptomycin, primary assay of StrpR; editing template chosen confers StrpR, has a mutation in the seed sequence to prevent incorporated PAM sequence from being targeted by CRISPR
    - we can prevent PAM recognition after incorporation by either excluding PAM or modifying the seed sequence near the PAM to prevent binding 
  - electroporation plus addition of edting template kills the cell unless recombination occurs, select for StrpR
  - any cell that has taken up the plasmid will have KanR, only those that have StrpR underwent recombination with plasmid
  - no targeting CRISPR has no strepR, while targeting CRISPR has strepR frequency of $1 \times 10^{-7}$
  - dummy CRISPR produces $8 \times 10^{-2}$ kanR frequency, while targeting CRISPR produces $1.4\times 10^{-5}$ kanR frequency
    - electroporation can introduce plasmid into 1 in 10 cells, but the targeting CRISPR kills some of the cells it is introduced to
    - escapees seem to outnumber editees by a large factor; editing is on par with random mutation, and large number of escapees indicate that CRISPR is not a suitable editing technology 
- in order to effectively use CRISPR in E. Coli, we must create recombinering E. Coli; E. Coli that can easily be engineered at multiple genomic locations, introduction of a more homologous recombination system (borrowed from phage, commonly used $\lambda$ phage's red recombination system)
  - includes a method for inhibiting endogenous recombination systems
  - recombinering E. Coli shows higher recombination frequency than prior (100-1000 fold), slight increase in recombination for targeting CRISPR (3-10 fold)
    - still significantly worse than Pneumococcus, but significantly better than non-recombineering E. Coli
  - adding CRISPR to recombineering colonies, one in 2-3 colonies appear to be editing by the targeting CRISPR system
  - editing frequency for targeting CRISPR is justified by killing of unedited cells, but non-targeting CRISPR also shows an increase in edting frequency 
    - we don't know the full answer, but likely has to do with the tendency of competent cells to take up and recombine compared to baseline
- dummy CRISPR doesn't show any difference in number of transformants across editing template
  - targeting CRISPR shows lower number of cells when editing template is not present; non-zero number of recombinants when editing template is not present suggests 1 in 2000 escapee frequency
- presence of kanR and strepR frequency shows a 6-10 fold increase in homologous recombination frequency for targeting CRISPR on recombineering E. Coli
  - selection mechanism is primarily killing of unedited colonies, but effect of double-strand cut stimulated homologous recombination is more significant in E. Coli

#### 11/16/2020
- Long et al. CRISPR to correct muscular dystrophy
  - Duchenne muscular dystropy: x-linked disease, mutations in the dystrophin gene result in muscle and organ degradtion; no treatment, 20-30 year life span for affected boys
  - interested in using CRISPR to fix stop codon introduced into large dystrophin gene
  - double-strand breaks in eukaryotes is more likely to result in NHEJ, introduction of errors and nonfunctional proteins
    - NHEJ may be useful in correcting Duchenne muscular dystrophy by shifting stop codon in and out of frame, could result in conserved protein structure
  - multiple exons are processed by splicing into a single protein; exon protein has a mutation from 5'CAA3' to 5'TAA3'
  - PAM (TGG) was found directly next to the target sequence, with editing template modifying several bases in the seed sequence (TseI restriction enzyme can be used on the introduced sites to determine if editing has occurred; prevents CRISPR hybridization since PAM codon is part of the functional Dys protein)
  - primer amplification of segment of interest can show either HDR or NHEJ (only HDR shows a corrected gene that can be cut with TseI, NHEJ may be able to be cut with TseI)
  - mdx zygote is formed from mdx/Y and mdx/mdx donors, then injected with Cas9 + sgRNA + ssODN and implanted to produce corrected mice if repair did occur
  - CRISPR was tried on wild-type mouse, editing was assayed with restriction site and sequencing s
    - wild-type pups show most bands of same size (1 is slightly larger, indicating NHEJ); but most show no cutting by TseI (3 of 17 show varying levels of cutting success), indicating that DNA might not be edited 
    - editing is possible, but variable between zygotes (ranges between strong-medium-weak-no editing)
    - zygote may have CRISPR working in certain cells, creation of mosaic organisms 
    - by mixing original and new loci, we can recognize mismatches by denaturing and annealing; small number of base mismatches produce looped out segments, which can be tagged by T7 endonuclease 
      - gel after T7 digestion shows that while alot of CRISPR repair did occur, most of it was repaired by NHEJ; CRISPR acts efficiently, NHEJ is just more common than HDR in eukaryotes 
      - HDR sequencing shows homologous DNA repair at most positions (we see mixed bases at the 4 seed sequence modifications, indicating that incomplete HDR has occurred; unedited cells give original sequence)
      - NHEJ shows lack of edits to seed sequence, after Cas9 cutting the sequence is random (insertions and deletions occur differently in cells as a result of NHEJ)
  - library can be cloned into a plasmid vector, and then conferred to bacterial colonies; each colony represents a single repair event, which can then be selected for and further analyzed

#### 11/18/2020
- sequences clones showed the following patterns; differing patterns of insertions and deletions owe to NHEJ and mosaic organisms 
  - 2x WT + 5x HDR
  - 10x WT + 5x IF+45ins
  - 8x WT + 2x -4del + 3x -5del + 1x +2ins
  - 9x WT + 2x IF-3del + 2x point mutations
- using the mdx zygotes, we can run gels of undigested, +TseI, +T7E1 to determine how much NHEJ is occuring
  - corrected samples have TseI restriction, and T7E1 determination of mismatches; complete correction should show complete digestion by TseI and no evidence of NHEJ by +T7E1 digestion
  - possibly corrected mice show variable proportions of HDR and unedited DNA; ranges from 1:23 to 6:0, seems like HDR partially or completely repairs the sequences
  - wide amount of variation in NHEJ-mediated corrections, includes large deletions and point mutations, also in variable frequency compared to wild 
- we can also amplify DNA in off-targets to determine if any inaccurate or unspecific changes CRISPR has (should be unchanged with a perfectly targeting CRISPR system)
  - in target, mdx and wild type controls both show low frequency of HDR and NHEJ
    - mdx + Cas9 shows mostly HDR, but WT + Cas9 shows same frequency of HDR to mdx, but much higher frequency of NHEJ
  - most off-target loci show no effect, some have small offtargets effects when comparing Cas9 samples to control samples 
- we can enzyme assay for creatine kinase in blood (high in muscular dystropy samples)
  - enzyme activity in most repair samples have improvements in CK blood concentration
  - even the NHEJ 83% pup saw a more significant repair to the disease than the fully HDR pup
- we can also test mouse muscle strength 
  - mice that had been corrected showed higher muscle strength levels (near wild-type measurements)
  - indicates that 100% correction is not needed to obtain good level of restoration of wild-type enzyme concentration and strength
  - 17% HDR, 41% HDR, and 53% NHEJ corrections all show improvements to CK blood concentration, muscular strength, and histopathology of muscle tissues 
  - muscle strength was found to show progressive improvements as the mice aged; explained by multinucleated satellite muscle cells 
    - fusion produces corrected, uncorrected, and mixed tissue nuclei; adequate amount of dystrophin can be made by mixed tissue, as mice age the corrected nuclei mix with previously uncorrected tissues 

#### Section 4 Readings

Jiang et al. paper

- CRISPR experiments carried out in Streptococcus pneumonia (known to be competent for DNA uptake; also was first experimental organism to show DNA as genetic material)
- first set of experiments: CRISPR-Cas9 introduction to cells with a susceptible targeting locus causes death; some escapers occurred, where susceptible locus was replaced by wild type locus from transforming DNA
  - gene replacement was enhanced by the presence of the CRISPR-Cas9 
- PAM sequence was randomized, then used to determine the resistance to CRISPR editing; analysis shows PAM of 5'NNG3'
  - similar randomization performed on the seed sequence: first 3-5 PAM proximal positions found to require complementarity to crRNA
- sequential loci editing + simultaneous multi loci editing shown in next experiment set
- in less robust homologous recombination bacteria (E. Coli or Salmonella), CRISPR in basic form is less useful
  - editing frequency compared to escape frequency is not sufficient
  - homologous recombination systems can be improved by generating recombineeering strains with phage lambda 'red' system
  
- Experiment 1
  - null hypothesis: when genome has appropriate target DNA, CRISPR-Cas9 will cleave the target, which will result in cell lethality
  - cell line with appropriate target shows transformants, but somewhat less efficient than cell line without appropriate target site
    - transformants from susceptible cell line arose by transformation to wild-type
  - conclusion: CRISPR promotes editing of genome via replacement by homologous repair from external source
- Experiment 2
  - will increasing the frequency of editing cassette in transforming DNA increase recombination frequency?
- Experiment 3
  - how to optimize PAM sequence for editing a genome?
  - consensus for PAM is 5'NGG3'
- Experiment 4
  - what requirements exist for the seed-sequence?
  - protospacer seems to have high degeneracy at most points, easier to mutate PAM + first 2/3 seed sequence positions
- Experiment 5
  - engineer strain to which spacers can be easily introduced 
  - use of Gibson assembly + directional PCR to create a cassette with spacer and antibiotic marker
  - editing DNA is introduced to another strain, 2 recombination events
    - introduce spacer and new antibiotic marker to target 
    - replace edited gene where target parental gene is 
- Experiment 6
  - test practicality of genome editing
  - mutate endogenous LacZ gene (easily assayed) with 3 types of editing cassette
    - inactivation of active site residue (modification of seed sequence)
    - two mutations inactivate the enzyme (PAM sequence modification)
    - large deletion creates mini-LacZ which is inactive
  - compared to wild type LacZ editor, modified editors produced 10 fold more transformants
  - transformant lines showed no noticeable LacZ enzyme activity 
- Experiment 7
  - test sequential editing of genes
  - by switching back and forth for drug markers, we can perform sequential editing
  - first experiment modifies srtA gene, results in LacZ escaping from cell surface into supernatant
  - next experiment modifies LacZ, double deletion mutants show no cellular LacZ activity
- Experiment 8
  - test simultaneous editing of multiple genes
  - introduce spacers and editing cassettes for both srtA and bgaA in a single step + kanR
  - majority of the transformants show the double deletion, though a couple show single deletions only
- Experiment 9
 - background level of editing, limitations of method
 - assay for editing is restoration of wild type ermAM; currently has ermAM(stop) which has an artificial PAM
  - expect that KanR transformants will be ErmR as well
  - first use non-targeting CRISPR
    - look for ermR colonies first without selecting for kanR: $10^{-2}$ of total colonies
    - look for ermR colonies as a fraction of KanR colonies: $10^{-1}$ of kanR colonies
  - now use targeting CRISPR
    - frequency of KanR transformants that are also ermR is 99%; unedited ermAM(Stop) gene is targeted by CRISPR which kills it
      - by measuring fraction of KanR produced with each CRISPR targeting system: functional CRISPR results in 5 to 6 times more KanR colonies than nonfunctional; CRISPR causes lethality to cells containing target DNA unless target gene is edited
      - not all unedited cells are killed; potentially due to CRISPR inactiving mutation or escaper mutation in target strain
        - test escaper frequency by applying both CRISPR systems without editing strain: $3\times10^{-3}$ escaper frequency
      - CRISPR found to have only a modest effect on recombination mediated repair (repair strain + either CRISPR type only shows 2-3 times difference in ermR + kanR colonies)
- Experiment 10
  - usage of CRISPR systems in less strongly recombinant bacteria
  - use of CRISPR Cas systems in plasmids with chlorampehnicol resistance, along with targeting sequence on a separate plasmid (kanR)
  - transformation of E. coli containing tracR-Cas9 plasmid with spacer plasmid, selection for srmR
  - lots of KanR cells escape CRISPR action even though they remain unedited; if edited gene can't be selected for, screening for drug transformants only is not effective
- Experiment 11
  - increase bacterial editing frequency
  - use of recombineering E. Coli strains: express phage lambda 'red' recombination system
    - targeting CRISPR produces 10 times more recombinants than nontargeting CRISPR as a fraction of all cells
    - nearly two-thirds of kanR transformants also contain the edited gene
  - both in the presence and absence of editing oligo, targeting CRISPR has higher proportion of kanR cells
    - CRISPR kills cells with unedited rpsL, driving fraction of edited cells up
    - some kanR cells survive targeting CRISPR by escaping, mutation or other CRISPR resistance gain
    - roughly 5 times more escapees than editees (of kanR, 4/5 are unedited, 1/5 is edited) in theory
      - since we saw 65% of kanR cells to be edited, there is some unaccounted for effect boosting selection
      - CRISPR might boost recombination 
        - verified to be a roughly 6 fold increase in recombination; possibly due to CRISPR assisted DNA breaks

















